esp/services/ws_dfu/ws_dfuService.cpp

```diff
@@ -6030,7 +6030,9 @@ void CWsDfuEx::getFileAccess(IEspContext &context, IUserDescriptor *udesc, SecAc
     bool writePermissions = (accessType == SecAccess_Write) || (accessType == SecAccess_Full);
     bool readPermissions = true; // by implication
 
-    StringBuffer fileName = req.getName();
+    CDfsLogicalFileName lfn;
+    lfn.set(req.getName());
+    StringBuffer fileName = lfn.get();
```

Comment:
no need to copy into a StringBuffer here, better to have const char *fileName = lfn.get();

Reply:
I may need to append the cluster name into the StringBuffer (please see the next 2 lines).

Reply:
right, I missed that, fine as is then.